Sources/SwiftTUI/View/Views/Stacks/ZStack.swift

```diff
@@ -0,0 +1,116 @@
+import Foundation
+
+public struct ZStack<Content: View>: View, Primitive, LayoutRoot {
+    public let content: Content
+    let alignment: Alignment
+
+    public init(alignment: Alignment = .center, @ViewBuilder _ content: () -> Content) {
```

Comment:
For `VStack` and `HStack` I've chosen a default alignment of top-leading (unlike SwiftUI) as I expected this would be more common and desired for text-based interfaces. I'd suggest we use that here as well, until a decision is made to change to a default center alignment for all views.